help/db_meeting-data.md

```diff
@@ -0,0 +1,39 @@
+### Zawartość tabel
+
+##### Dictionary -- nowe słowniki
+
+| column                   | dictionary_id                        |
+|--------------------------|--------------------------------------|
+| MeetingType.category     | ce12aa23-a5db-49f3-987b-d2db3ab24a3b |
+| Meeting.status           | 3865a3c3-0038-4668-9d55-5d05b79d7fcd |
+| MeetingResource.resource | fdb0f818-aa1e-4ed9-97cc-2a3cb1b702df |
+
+##### MeetingType
+
+| id | facility_id | name               | category_dict_id | default_minutes |
+|----|-------------|--------------------|------------------|-----------------|
+| 1  | 1           | Konsultacja prawna | 1                | 60              |
+| 2  |             | Konferencja        | 2                | 180             |
+
+##### Meeting
+
+| id | facility_in | meeting_type_id | name                  | notes            | date       | start_minutes | minutes | status_dict_id |
+|----|-------------|-----------------|-----------------------|------------------|------------|---------------|---------|----------------|
+| 1  | 1           | 1               |                       | Dłuższy opis ... | 2023-08-06 | 600 (10:00)   | 60      | 1              |
+| 2  | 1           | 2               | Podsumowanie miesiąca |                  | 2023-19-15 | 700 (11:40)   | 150     | 2              |
+
+##### MeetingAttendant
+
+attendance_type - enum: client|staff|other
+
+| id | meeting_id | user_id | attendance_type | client_family_id | has_arrived |
+|----|------------|---------|-----------------|------------------|-------------|
+| 1  | 1          | 1       | client          | 1                | true        |
+| 2  | 1          | 2       | staff           |                  | false       |
+
+##### MeetingResource
```

Comment:
A to jest co w ogóle? Załączniki jakieś do spotkania? Bo jak tak to attachment raczej, słowo resource już jest za bardzo zjechane. A jak nie to nie rozumiem - mogłoby być napisane po ludzku.

Reply:
prosto z versum, uniwersalny mechanizm do rezerwacji sal, rzutników i innych losowych rzeczy
![image](https://github.com/mblajek/Memo/assets/4807088/ec738473-a7be-4ada-a39b-8616d5b6ef9f)

Reply:
nawet można po tym wyświetlać kolumny na kalendarzu